charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -134,7 +134,7 @@ spec:
               fieldRef:
                 fieldPath: status.hostIP
           - name: ALLUXIO_CLIENT_JAVA_OPTS
-            value: " -Dalluxio.user.hostname=${ALLUXIO_CLIENT_HOSTNAME} "
+            value: " -Dalluxio.user.hostname=${ALLUXIO_CLIENT_HOSTNAME} -Dalluxio.user.metadata.cache.enabled=false "
```

Comment:
Though closing the cache can let the fuse client update immediately, it can also make loss of profermance.
I suggest not to edit it.
If user want to update the mount points in fuse client, they can restart it.

Reply:
I suggest not make this hard code.

Reply:
ok, will remove it

Reply:
@yangyuliufeng restart what? : )

Reply:
Thanks, just ignore it.